### ISSUE TITLE
Bots Ghosting

### ISSUE THREAD

**Issue by braindisconnect** [EQEmu/Server#810]
SendPosition() was removed a couple of months ago.  adding in SentPositionPacket will fix.

Also bots are "rubber banding" when moving.  I suspect there is a fix similar to the ghosting fix.

Line 1830 add SentPositionPacket(0.0f, 0.0f, 0.0f, 0.0f, 0);

                //6 seconds, or whatever the rule is set to has passed, send this position to everyone to avoid ghosting
                if(!IsMoving() && !IsEngaged()) {
                      +SentPositionPacket(0.0f, 0.0f, 0.0f, 0.0f, 0);
                        if(IsSitting()) {
                                if(!rest_timer.Enabled())
                                        rest_timer.Start(RuleI(Character, RestRegenTimeToActivate) * 1000);
                        }
                }

**Comment by Akkadius** [comment]
Yeah bots were not taken into consideration with the recent movement overhaul, they need a little love to get up to par. We will take a look soon
